modes/debug_mode/debug_plot_page.py

```diff
@@ -56,10 +58,10 @@ def __init__(self, key_value: DebugPlotValue, parent: Frame):
 class DebugPlot(Page):
     def __init__(self, parent: Frame, model: Model):
         super().__init__(parent, model, "Debug Plot")
-        self.data: Dict[int, DebugPlotValue] = model.pinned_data
         self.time_presets = [30, 60, 120, 300, 600]
         self.current_time_index = 0
         self.current_time = self.time_presets[self.current_time_index]
+        self.data: Dict[int, DebugPlotValue] = self.model.pinned_data
```

Comment:
why is it called DebugPlotValue if it contains a list of values? should be `DebugPlotLineData` or something